site/content/en/docs/_index.md

```diff
@@ -1,15 +0,0 @@
-+++
```

Comment:
why are these removed ?

Reply:
we didn't nee these, make docs will copy them when release doc site.

Reply:
will the Hugo "Edit this Page" feature still work ?

Reply:
I think it won't.

Reply:
I recall that ths is added by @zhaohuabing, for accessing default path as the latest release version. Need a review from @zhaohuabing

Reply:
If we could point the "Edit this Page" links in the docs to the latest released version (e.g. v1.3), then it would work.

Reply:
is that possible using hugo config @zhaohuabing @zirain  ? if so then this change is a great one

Reply:
I'm not sure about it, I'm not a fan of that, will take a look.

Reply:
seems https://www.docsy.dev/docs/adding-content/repository-links/#path_base_for_github_subdir-optional may help.